PWGHF/TableProducer/candidateCreatorToXiPi.cxx

```diff
@@ -88,10 +88,9 @@ struct HfCandidateCreatorToXiPi {
   using FilteredHfTrackAssocSel = soa::Filtered<soa::Join<aod::TrackAssoc, aod::HfSelTrack>>;
   using MyCascTable = soa::Join<aod::CascDatas, aod::CascCovs>; // to use strangeness tracking, use aod::TraCascDatas instead of aod::CascDatas
   using MyV0Table = soa::Join<aod::V0Datas, aod::V0Covs>;
-  using MySkimIdx = soa::Filtered<HfCascLf2Prongs>;
+  using MySkimIdx = HfCascLf2Prongs;
 
   Filter filterSelectCollisions = (aod::hf_sel_collision::whyRejectColl == 0); // filter to use only HF selected collisions
-  Filter filterSelectIndexes = (aod::hf_track_index::hfflag == 1);
```

Comment:
Have you tried to do a type cast, like

`Filter filterSelectIndexes = (aod::hf_track_index::hfflag == (uint8_t) 1);`  ?

This is done in some tasks, e.g. here https://github.com/AliceO2Group/O2Physics/blob/master/PWGLF/Tasks/mcspectraefficiency.cxx#L106

Reply:
Thanks @mfaggin, I have changed to this version